### DISCUSSION TITLE
Moving from  Java32bit VM to 64 vbit

### DISCUSSION THREAD

**Discussion by azdave2020** [nextgenhealthcare/connect#5306] [Q&A]
For reasons that are moot, our Mirth install was done with a 32-bit VM.
I would like to install the 64-bit VM and point mirth to it.
What are the risk and how do I change the pointer?
Dave
Mirth Connect Server 3.10.1

Built on January 6, 2021

Server ID: e05420cc-68c1-4246-86fd-e0e17c5e16c5

Java version: 1.8.0_331

Running on windows server 2019 32GB ram

**Comment by jonbartels** [comment]
I think the risks are generally low. 

In an ideal world moving from one JDK/JRE to another should "just work" because thats the promise of the JVM. 

Your most likely risk is if you change JDK versions at the same time. At which point you should read the "Java 9" section of the MC admin guide and apply the correct settings. 

This thread should answer your questions about how to select a different JAVA_HOME for MC - https://forums.mirthproject.io/forum/mirth-connect/support/180215-specify-which-java-version-the-mirth-service-will-use-in-windows
